### PR TITLE
Issue #13213: Remove '//ok' comments from input files

### PR DESCRIPTION
**Pull request by yousifelboukhary**
removed //ok comment from InputJavadocMethodReceiverParameter.java, InputJavadocMethodScopeAnonInner.java

### CODE REVIEW

File: config/checkstyle-input-suppressions.xml [hunk 1]
```diff
@@ -597,16 +597,6 @@
             files="checks[\\/]javadoc[\\/]javadocmethod[\\/]InputJavadocMethodPublicOnly1.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]javadoc[\\/]javadocmethod[\\/]InputJavadocMethodReceiverParameter.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]javadoc[\\/]javadocmethod[\\/]InputJavadocMethodReceiverParameter.java"/>
```
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:601]: Looks like you have missed removal of suppression here. File `InputJavadocMethodReceiverParameter.java` is still being suppressed.

**Review by MANISH-K-07 — COMMENTED** [review]
@yousifelboukhary ,
It is generally not recommended to close and open multiple new PRs for the same changes.
All actions and modifications can be performed through git cmd.

File: config/checkstyle-input-suppressions.xml (hunk 1 above)
Comment (yousifelboukhary) [reply config/checkstyle-input-suppressions.xml:601]: @MANISH-K-07 I solved the problem but It gives me "This branch cannot be rebased due to conflicts"

**Comment by romani** [comment]
Single commit please

File: config/checkstyle-input-suppressions.xml (hunk 1 above)
Comment (MANISH-K-07) [reply config/checkstyle-input-suppressions.xml:601]: That must be because the changes you have made have already been done and merged in any other PR

**Comment by romani** [comment]
We lost contact with contributor

**Pull request closed without merging**